status/flowcells.py

```diff
@@ -104,13 +105,25 @@ def list_flowcells(self, all=False):
             # NovaSeqXPlus has whole year in name
             if "LH" in row.value["instrument"]:
                 note_key = f'{row.value["run id"].split("_")[0]}_{row.value["run id"].split("_")[-1]}'
-            latest_note = LatestRunningNoteHandler.get_latest_running_note(
-                self.application, "flowcell", note_key
-            )
-            if latest_note:
-                row.value["Latest Workset Notes"] = latest_note
+            note_keys.append(note_key)
             temp_flowcells[row.key] = row.value
 
+        notes = self.application.running_notes_db.view(
+            "latest_note_previews/flowcell",
+            reduce=True,
+            group=True,
+            keys=list(note_keys),
+        )
+        for row in notes:
+            key = row.key
+            if len(row.key.split("_")[0]) > 6:
```

Comment:
I would add a comment here of what's going on...

Reply:
Done!